components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -790,7 +790,7 @@ class RewardsServiceImpl : public RewardsService,
   bool reset_states_;
   bool ledger_for_testing_ = false;
   bool resetting_rewards_ = false;
-  bool should_persist_logs_ = false;
+  int persist_log_level_ = 0;
```

Comment:
`0` will only log errors. Ideally, this should be, which is classified as `INFO`

Reply:
I agree that more logging would be ideal. Unfortunately, logging at 2 will create for us a dilemma: we'll likely have to *remove* useful information from messages at those levels, which will make those levels less useful for development and detailed debugging. I would prefer that we stay with logging only errors by default in this PR, and consider increasing the default log level as a future change.